content/en/bits_ai/mcp_server/setup/_index.md

```diff
@@ -0,0 +1,331 @@
+| [Goose](https://github.com/block/goose) | Block | |
+| [Q CLI](https://github.com/aws/amazon-q-developer-cli) | Amazon | Does not support HTTP transport for remote authentication. Use [local binary authentication](#connect-to-the-datadog-mcp-server) instead. |
+| [VS Code](https://code.visualstudio.com/) | Microsoft | Datadog [VS Code & Cursor extension](#connect-in-vs-code-or-cursor) recommended. |
+
+### Connect in VS Code or Cursor
+
+Datadog's [VS Code and Cursor extension][1] includes built-in access to the Datadog MCP Server. Benefits include:
+
+* No additional MCP Server setup after you install the extension and connect to Datadog.
+* One-click transitions between multiple Datadog organizations.
+* Better fixes from **Fix in Chat** on Code Insights (issues from Error Tracking, Code Vulnerabilities, and Library Vulnerabilities), informed by context from the MCP Server.
+
+To install the extension:
+
+1. Ensure you have an active version of [Node.js][2] installed on your local machine.  
+2. If you previously installed the Datadog MCP Server manually, remove it from the IDE's configuration to avoid conflicts. To find the MCP Server configuration:
+   - VS Code: Open the command palette (`Shift` + `Cmd/Ctrl` + `P`) and run `MCP: Open User Configuration`.
+   - Cursor: Go to **Cursor Settings** (`Shift` + `Cmd/Ctrl` + `J`) and select the **Tools & Integrations** tab.
+3. Install the Datadog extension following [these instructions][3]. If you have the extension installed already, make sure it's the latest version, as new features are released regularly.  
+4. Sign in to your Datadog account. If you have multiple accounts, use the account included in your Product Preview.
+    {{< img src="bits_ai/mcp_server/ide_sign_in.png" alt="Sign in to Datadog from the IDE extension" style="width:70%;" >}}
+5. Open a project or workspace.
+6. **Restart the IDE.**
+7. Confirm the Datadog MCP Server is available and the [tools](#available-tools) are listed in your IDE:
+    - VS Code: Open the chat panel, select Agent mode, and click the **Configure Tools** button.
+       {{< img src="bits_ai/mcp_server/vscode_configure_tools_button.png" alt="Configure Tools button in VS Code" style="width:70%;" >}}
+    - Cursor: Go to **Cursor Settings** (`Shift` + `Cmd/Ctrl` + `J`), and select the **Tools & Integrations** tab.
+
+## Available tools
+
+This section lists the tools available in the Datadog MCP Server and provides example prompts for using them.
+
+<div class="alert alert-warning">Datadog MCP Server tools are under significant development and are subject to change. We encourage you to share any feedback, use cases, or issues encountered with your prompts and queries.</div>
+
+### `ask_docs`
+Returns AI-generated answers to Datadog questions, sourced from [Datadog documentation][4].
+- "How do you enable Datadog profiling in Python?"
```

Comment:
Would you consider formatting all of these lists without the quotation marks? I think it may look better and still be understandable. It's a matter of opinion, though, so please go with what you prefer.

Reply:
I tried it out on my local build, but to me it looked weird to have questions without punctuation. Only a few of the prompts are questions, so it doesn't affect most of the examples.

Reply:
Sorry, let me be more specific. I wasn't referring to the question marks. I meant the quotes. See the suggestion text below.
```suggestion
- How do you enable Datadog profiling in Python?
```

Reply:
🤦 Oh wow, sorry I totally misread that! No quotation marks sounds promising; I'll try it out...